docs/development/codeguide.rst

```diff
@@ -17,13 +17,6 @@ Interface and Dependencies
   <https://github.com/astropy/astropy/blob/main/pyproject.toml>`_ file of the
   core package.
 
-* Usage of ``six`` and ``2to3`` is no longer acceptable.
```

Comment:
If this was something worth worrying about then we should enforce it with [TID251 (banned-api)](https://docs.astral.sh/ruff/rules/banned-api/), but I don't think it's needed.